<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>441</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore to Defaults</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="Video">
    <attribute name="title">
     <string>Video</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>331</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>331</height>
       </rect>
      </property>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>381</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="formAlignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>FPS</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_2"/>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Use VSync</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Controls">
    <attribute name="title">
     <string>Controls</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>331</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>331</height>
       </rect>
      </property>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>10</y>
         <width>381</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="formAlignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Drag Delay</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Key Repeat Delay</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Key Interval Delay</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_8"/>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBox_9"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Grid">
    <attribute name="title">
     <string>Grid</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>331</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>331</height>
       </rect>
      </property>
      <widget class="QWidget" name="formLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>381</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="formAlignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Dot Size</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_3"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Dot Spread</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_4"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Focus Size</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBox_6"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Line Thickness</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBox_7"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Offscreen Amount</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="spinBox_10"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Colors">
    <attribute name="title">
     <string>Colors</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>331</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>399</width>
        <height>619</height>
       </rect>
      </property>
      <widget class="QWidget" name="formLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>361</width>
         <height>591</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <property name="formAlignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Background</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Bounds Circles</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Bounds Lines</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Dots</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Focus</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Toolbar Colors</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="ColorDialog" name="pushButton_2">
          <property name="text">
           <string/>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="ColorDialog" name="pushButton_4">
          <property name="text">
           <string/>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="ColorDialog" name="pushButton_5">
          <property name="text">
           <string/>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="ColorDialog" name="pushButton_6">
          <property name="text">
           <string/>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <layout class="QFormLayout" name="formLayout_10">
          <item row="0" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ColorDialog" name="pushButton_7">
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ColorDialog" name="pushButton_8">
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="ColorDialog" name="pushButton_9">
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="ColorDialog" name="pushButton_10">
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="ColorDialog" name="pushButton_11">
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="ColorDialog" name="pushButton_12">
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="ColorDialog" name="pushButton_13">
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="ColorDialog" name="pushButton_14">
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="ColorDialog" name="pushButton_15">
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="ColorDialog" name="pushButton_16">
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="ColorDialog" name="pushButton_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Save">
    <attribute name="title">
     <string>Saving</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>331</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>331</height>
       </rect>
      </property>
      <widget class="QWidget" name="formLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>381</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_5">
        <property name="formAlignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Export Line Thickness</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_5"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Default Save Location</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="OpenFileDialog" name="lineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Default Export Location</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="OpenFileDialog" name="lineEdit_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorDialog</class>
   <extends>QPushButton</extends>
   <header location="global">ColorDialog</header>
  </customwidget>
  <customwidget>
   <class>OpenFileDialog</class>
   <extends>QLineEdit</extends>
   <header location="global">ColorDialog</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>pressed()</signal>
   <receiver>pushButton_2</receiver>
   <slot>showMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
